app/controllers/updateRegisteredDetails/UpdateDoneController.scala

```diff
@@ -40,7 +41,11 @@ class UpdateDoneController @Inject()(
   def onPageLoad: Action[AnyContent] = controllerActions.withRequiredJourneyData(UpdateRegisteredDetails) {
 
     implicit request =>
-      val summaryList = Seq(UpdateContactDetailsSummary.rows(request.userAnswers)).flatten
+      val ukSiteDetailsSummary: Option[(String, SummaryList)] = UKSitesSummary.getHeadingAndSummary(request.userAnswers, isCheckAnswers = false)
+      val updateContactDetailsSummary: Option[(String, SummaryList)] = UpdateContactDetailsSummary.rows(request.userAnswers, isCheckAnswers = false)
+      val businessAddressSummary: Option[(String, SummaryList)] = BusinessAddressSummary.rows(request.userAnswers, isCheckAnswers = false)
+      val summaryList = Seq(ukSiteDetailsSummary, updateContactDetailsSummary, businessAddressSummary).flatten
+
       val getSentDateTime = LocalDateTime.now(ZoneId.of("UTC")) //LocalDateTime.ofInstant(request.userAnswers.submittedOn.get, ZoneId.of("UTC"))
```

Comment:
The getSentDateTime should come from the user answers, submitted, however, a ticket has been raised for this issue on all journeys. No change required on this PR. /